projects/plugins/boost/tests/e2e/package.json

```
@@ -14,18 +14,19 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
+		"build": "pnpx jetpack build plugins/jetpack --production && pnpx jetpack build plugins/boost -v --production",
```

As of #22538, you can now do
```suggestion
		"build": "pnpx jetpack build plugins/jetpack plugins/boost -v --no-pnpm-install --production",
```